### PR TITLE
Update version: RoyalApps.RoyalTS.V7 version 7.02.50424.0

### LINKED ISSUE #151770: [Update Request]: RoyalTS V7 7.02.50424
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

RoyalApps.RoyalTS.V7

### Package Version

7.02.50424

### Please describe the changes you would like to see

_No response_

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] This resolves #151770

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151787)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @Dragon1573,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 878.)

File: manifests/r/RoyalApps/RoyalTS/V7/7.02.50424.0/RoyalApps.RoyalTS.V7.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS.V7
+PackageVersion: 7.02.50424.0
+PackageLocale: en-US
+Publisher: Royal Apps GmbH
+PublisherUrl: https://www.royalapps.com/
+PublisherSupportUrl: https://support.royalapps.com/support/home
+PackageName: Royal TS 7.02.50410.0
```
Comment (Trenly) [inline manifests/r/RoyalApps/RoyalTS/V7/7.02.50424.0/RoyalApps.RoyalTS.V7.locale.en-US.yaml:10]:
```suggestion
PackageName: Royal TS V7
```

File: manifests/r/RoyalApps/RoyalTS/V7/7.02.50424.0/RoyalApps.RoyalTS.V7.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+PackageVersion: 7.02.50424.0
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+ProductCode: '{4959A733-614C-44E8-AC0D-3E7BCAA2F5AC}'
+ReleaseDate: 2024-04-24
+AppsAndFeaturesEntries:
+- DisplayName: Royal TS V7
+  DisplayVersion: 7.2.50424.0
+  UpgradeCode: '{4959A733-614C-44E8-AC0D-3E7B5A7375AB}'
```
Comment (Trenly) [inline manifests/r/RoyalApps/RoyalTS/V7/7.02.50424.0/RoyalApps.RoyalTS.V7.installer.yaml:14]:
```suggestion
- UpgradeCode: '{4959A733-614C-44E8-AC0D-3E7B5A7375AB}'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Hi @Dragon1573,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 880.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**